controllers/logging_client.go

```diff
@@ -0,0 +1,313 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !release
+
+package controllers
```

Comment:
isn't `integrationtests` better place for this file? or some other test package?

Reply:
My original intention was to also use this outside of integration tests but since currently it is only being used there and frankly there is little incentive to use it in normal unit tests that have much tighter control over the K8s calls, I think you're right. I moved it to `integrationtests` package.